conf/schemas.go

```diff
@@ -8,20 +8,21 @@ import (
 	"strings"
 
 	"github.com/alyu/configparser"
+	"github.com/raintank/metrictank/util"
 )
 
-var DefaultSchema = Schema{
-	Name:       "default",
-	Pattern:    nil,
-	Retentions: Retentions([]Retention{NewRetentionMT(60, 3600*24*7, 120*60, 2, true)}),
+// Schemas contains schema settings
+type Schemas struct {
+	raw           []Schema
+	index         []*Schema
```

Comment:
at one stage match was performing a range over the index.  A range of a slice copies the values, so only copying the reference is much faster then copying the whole schema struct.

But doenst look like this is needed anymore